Source/Tests/LinuxSyscalls/FileManagement.cpp

```diff
@@ -313,6 +278,44 @@ FileManager::FileManager(FEXCore::Context::Context *ctx)
     }
   }
 
+  // Now that we loaded the thunks object, walk through and ensure dependencies are enabled as well.
+  for (auto const &DBObject : ThunkDB) {
+    if (DBObject.second.Enabled) {
+      auto ThunkPath = ThunkGuestPath / DBObject.second.LibraryName;
+      if (std::filesystem::exists(ThunkPath)) {
+        for (auto Overlay : DBObject.second.Overlays) {
```

Comment:
This avoids unnecessary copies of the contained `std::string`s:
```suggestion
        for (const auto& Overlay : DBObject.second.Overlays) {
```

Reply:
Resolved.